source/_patterns/00-atoms/10-misc/07-image-hero.mustache

```diff
@@ -0,0 +1,3 @@
+<div class="heroImageContainer ">
```

Comment:
Why is this file name backwards? :D
(_image-hero_ instead of _hero-image_)

Reply:
I see your point. I even saw it before pushing. The reason is, that i wanted it to appear as Image Hero in the Patternlab, cause the other images also appear as Image Whatever. :D But I could also change it.

Reply:
If there's a reason you should keep it as it is!

Reply:
Changed, cause it's actually not needed